schedule.md

```diff
@@ -0,0 +1,89 @@
+# Session plan
+
+## Session 1: R and statistical concepts used
+
+Some of this could be pre-course reading material, see e.g. https://sbfnk.github.io/mfiidd/slides/rintro.pdf
+
+Monday 9:30-11:00
+
+- Introduction to R concepts used in the course (15 mins)
```

Comment:
Can we really get away without a brief intro to stan itself?

Reply:
No, good point.

Reply:
Closing in favour of the near duplicate below and #21 and #22.